EnglishTranslation-master/ITEM.tsv

```diff
@@ -10305,24 +10305,24 @@ ITEM_20160310_010304	Experiment Herb Seeds
 ITEM_20160310_010305	Herb seeds Lucienne has asked you to plant. It seems they're going to be used to make a reagent.
 ITEM_20160310_010306	Soul Sphere
 ITEM_20160310_010307	A sphere given by a villager. It is said to prevent attacks when filled with demon souls.
-ITEM_20160310_010308	Red blood collected from a demon. It's impossible to know what it will be used for.
+ITEM_20160310_010308	Red blood collected from a demon. It's impossible to know what its will be used for.
```

Comment:
I originally changed this to "impossible to know what its uses will be," but I reverted it back and left a typo.  However, let me know what would be best, as I had trouble deciding between the two.